packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -67,11 +74,22 @@ void describe('getting started happy path', async () => {
   });
 
   void it('creates new project and deploy them without an error', async () => {
-    await runPackageManager(
-      packageManager,
-      ['create', 'amplify', '--yes'],
-      tempDir
-    ).run();
+    // TODO: remove the condition once GA https://github.com/aws-amplify/amplify-backend/issues/1013
+    if (packageManager === 'yarn-classic') {
+      await execa('yarn', ['global', 'add', 'create-amplify@beta']);
```

Comment:
does this have to be global?

Reply:
Nope. I'll change it.

Reply:
Yes, we have to use `global` here because `yarn create` uses `global` install 😭

Reply:
ah gotcha. Maybe leave a small comment explaining that